modules/oe_theme_helper/tests/src/Functional/PageHeaderBlockTest.php

```diff
@@ -66,20 +66,20 @@ public function testRendering(): void {
     $assert_session = $this->assertSession();
 
     // Only one page header should be rendered.
-    $assert_session->elementsCount('css', '.ecl-page-header-core', 1);
-    $header = $this->getSession()->getPage()->find('css', '.ecl-page-header-core');
+    $assert_session->elementsCount('css', '.ecl-page-header', 1);
+    $header = $this->getSession()->getPage()->find('css', '.ecl-page-header');
     // Test that the page title is rendered in the page header.
-    $this->assertEquals($entity->label(), trim($header->find('css', '.ecl-page-header-core__title')->getText()));
+    $this->assertEquals($entity->label(), trim($header->find('css', '.ecl-page-header__title')->getText()));
     // Intro and meta items are empty.
     $assert_session->elementsCount('css', '.ecl-page-header__intro', 0);
     $assert_session->elementsCount('css', '.ecl-meta--header .ecl-meta__item', 0);
 
     // Test another route.
     $this->drupalGet('/user/login');
-    $assert_session->elementsCount('css', '.ecl-page-header-core', 1);
-    $header = $this->getSession()->getPage()->find('css', '.ecl-page-header-core');
-    $this->assertEquals('Log in', trim($header->find('css', '.ecl-page-header-core__title')->getText()));
-    $assert_session->elementsCount('css', '.ecl-page-header-core__description', 0);
+    $assert_session->elementsCount('css', '.ecl-page-header', 1);
```

Comment:
The negative page header class should be present so please assert that.

Reply:
Ok